### PR TITLE
Fix : package.json / package-lock.json

### LINKED ISSUE #2932: 🐛 Bug Report: unwanted react-query
### What happened?

In the commit https://github.com/amplication/amplication/commit/6fe3e9522fe9306b3c10f0188f0683f7807cef5f we accidentally added the react-query package to the package.json

### What you expected to happen

not relevant

### How to reproduce

not relevant

### Amplication version

0.13.1

### Environment

_No response_

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by astitva0011**
Close #2932

## PR Details

    I have Fixed  The package.json and package-lock.json
## PR Checklist
- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,10 +22,9 @@
     "setup:dev": "ts-node ./script/setup.ts",
     "docker:dev": "docker-compose -f docker-compose.dev.yml up -d"
   },
-  "dependencies": {
+   "dependencies": {
```
Comment (abrl91) [inline package.json:25]: looking good! just undo this space please

Comment (astitva0011) [reply package.json:25]: space before last "dependencies":?

Comment (abrl91) [reply package.json:25]: yes :)

Comment (astitva0011) [reply package.json]: i have undo the space

Comment (astitva0011) [reply package.json]: why it is showing outdated?

**Review by abrl91 — APPROVED** [review]

**Comment by astitva0011** [comment]
Thanks for approving the changes 🥳

**Pull request merged**